CRM/Certificate/Service/CertificateDownloader.php

```diff
@@ -6,7 +6,7 @@
 
 class CRM_Certificate_Service_CertificateDownloader {
 
-  public int $format;
+  public? int $format;
```

Comment:
I think it looks more natural if you add the `?` before the type like this:

`
public ?int $format;
`